### PR TITLE
add auto-roll setting

### PR DESCRIPTION
**Pull request by toddmedema**
Closes https://github.com/ExpeditionRPG/expedition-app/issues/17

Screenshots:

screen shot 2017-03-27 at 10 47 41 pm
screen shot 2017-03-27 at 10 48 09 pm
screen shot 2017-03-27 at 10 41 07 pm

### CODE REVIEW

File: app/components/CombatContainer.tsx [hunk 1]
```diff
@@ -65,12 +65,12 @@ const mapDispatchToProps = (dispatch: Redux.Dispatch<any>, ownProps: any): Comba
       dispatch(combatDefeat());
     },
     onTimerStop: (cardName: CardName, elapsedMillis: number, settings: SettingsType, surge: boolean) => {
+      dispatch(handleCombatTimerStop(elapsedMillis, settings));
```
Comment (smartin015) [inline app/components/CombatContainer.tsx]: Why was this moved?

File: app/reducers/combat.tsx [hunk 2]
```diff
@@ -95,14 +95,23 @@ export function generateLoot(maxTier: number): Loot[] {
...
   return loot;
-};
+}
+
+function generateRolls(count: number): number[] {
+  const rolls = [];
+  for (let i = 0; i < count; i++) {
+    rolls.push(Math.floor(Math.random() * 20) + 1);
+  }
+console.log(count, rolls);
```
Comment (smartin015) [inline app/reducers/combat.tsx]: Extraneous console log

Comment (toddmedema) [inline app/reducers/combat.tsx]: fixed

File: app/components/CombatContainer.tsx (hunk 1 above)
Comment (smartin015) [inline app/components/CombatContainer.tsx]: Something to keep in mind is that toCard also checkpoints the history... if we weren't skipping timer cards, this would cause skew of e.g. roundCount (gets incremented, history checkpoint. User clicks back button to timer card, stops timer again and increments roundCount). 

I *think* this is OK since we're skipping over the timer card on back action, but I'd recommend double-checking to be sure.
